dashboard/src/components/cfp-public/SessionDetailForm.vue

```
@@ -4,14 +4,16 @@
       <IconClipboardText />
       <span>Submission Form</span>
     </h4>
+    <div v-for="(_field, index) in fields">
```

ESLint

[error] Elements in iteration expect to have 'v-bind:key' directives.

(vue/require-v-for-key)